phalcon/Assets/Collection.zep

```diff
@@ -71,7 +71,7 @@ class Collection implements Countable, Iterator
     /**
      * @var int
      */
-    protected position { get };
+    protected position = 0 { get };
```

Comment:
Looks like you solved a bug as well. Can you add an entry to the change log?

Reply:
It seems :-), @Jeckerson helps me to solve this problem. And ok about the change log.